prac_08/squaring.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+Kivy GUI program to square a number
+Lindsay Ward, IT@JCU
+Started 13/10/2015
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+__author__ = 'Lindsay Ward'
+
+
+class SquareNumberApp(App):
+    """ SquareNumberApp is a Kivy App for squaring a number """
```

Comment:
I know this is lindsay's fault but you it would help to fix his mistake and fix the docstrings, (capital at start and full stop no spaces)